<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>774</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutoTech - Cadastro</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/imagens/logo.png</normaloff>:/icon/imagens/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FF915D;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_error">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #170900;
border-radius: 7px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="text_error">
            <property name="font">
             <font>
              <family>Gilroy-Bold</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
color:#D9D9D9;
}
</string>
            </property>
            <property name="text">
             <string>Algum campo está inválido ou vazio</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="icon_error">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/imagens/imagens/x-circle-regular-24 (1).png);
background-position:center;
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="container_frame">
      <property name="styleSheet">
       <string notr="true">background-color: #FF915D;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="content_frame">
         <property name="maximumSize">
          <size>
           <width>550</width>
           <height>600</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#170900;
border-radius:10px;
box-shadow: 10px 10px 21px -7px rgba(0,0,0,0.61);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="logo">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>10</y>
            <width>210</width>
            <height>150</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/imagens/imagens/logo_250.png);
background-repeat: no-repeat;
background-position:center;
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_nome">
          <property name="geometry">
           <rect>
            <x>135</x>
            <y>200</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Gilroy-Regular</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	color: #AAAAAA;
	background-color: #170900;
	border:1px solid #A6A6A6;
}

QLineEdit:hover{
	border:1px solid #DADADA;
}

QLineEdit::focus{
	border: 2px solid #FFBD59;
}</string>
          </property>
          <property name="maxLength">
           <number>80</number>
          </property>
          <property name="placeholderText">
           <string>Nome Completo</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_cpf">
          <property name="geometry">
           <rect>
            <x>135</x>
            <y>260</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Gilroy-Regular</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	color: #AAAAAA;
	background-color: #170900;
	border:1px solid #A6A6A6;
}

QLineEdit:hover{
	border:1px solid #DADADA;
}

QLineEdit::focus{
	border: 2px solid #FFBD59;
}</string>
          </property>
          <property name="maxLength">
           <number>11</number>
          </property>
          <property name="placeholderText">
           <string>Cpf</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_senha">
          <property name="geometry">
           <rect>
            <x>135</x>
            <y>320</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Gilroy-Regular</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	color: #AAAAAA;
	background-color: #170900;
	border:1px solid #A6A6A6;
}

QLineEdit:hover{
	border:1px solid #DADADA;
}

QLineEdit::focus{
	border: 2px solid #FFBD59;
}</string>
          </property>
          <property name="maxLength">
           <number>16</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Senha</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>135</x>
            <y>460</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: transparent;
	border: 2px solid #FFBD59;
	border-radius: 25px;
	color: #FFBD59;
	font: 14pt &quot;Gilroy-Bold&quot;;
	transition:1s;
}

QPushButton:hover{
	background-color: #FFBD59;
	color: #170900;
}</string>
          </property>
          <property name="text">
           <string>Cadastrar</string>
          </property>
         </widget>
         <widget class="QFrame" name="frame_2">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>530</y>
            <width>330</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>330</width>
            <height>30</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>165</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
	color:#A6A6A6;
	font: 11pt &quot;Gilroy-Regular&quot;;
}</string>
             </property>
             <property name="text">
              <string>Já possui uma conta?</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_login">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>145</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Gilroy-Bold</family>
               <pointsize>11</pointsize>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background-color: transparent;
	border: 2px solid #FFBD59;
	border-radius: 10px;
	color: #FFBD59;
	font: 11pt &quot;Gilroy-Bold&quot;;
}

QPushButton:hover{
	background-color: #FFBD59;
	color: #170900;
}</string>
             </property>
             <property name="text">
              <string>Fazer Login</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QDateEdit" name="dataNascimento">
          <property name="geometry">
           <rect>
            <x>135</x>
            <y>380</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Gilroy-Medium&quot;;
color: #D9D9D9;
border: 1px solid #D9D9D9;</string>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="bottom_frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FF915D;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
